docs/optlib.rst

```diff
@@ -1425,14 +1412,13 @@ You can implement such roles by extending the built-in C parser:
 
 .. _guest-regex-flag:
 
-Running a guest parser with ``_guest`` regex flag
+Defining a guest parser with ``_guest`` regex flag
```

Comment:
How about `Scheduling running a parser as a guest parser` ?
`Defining` implies writing a new parser for me.

Reply:
I agree with you.  I've fixed to "Defining".

Reply:
I've fixed.